### PR TITLE
Add stderr redirection

### PR DESCRIPTION
**Pull request by c64cosmin**
It collects stdout and stderr of important regression test steps and prints them after the completion of the tests. It also writes all .log files generated by OMI server.

### CODE REVIEW

File: testcli/start_server [hunk 1]
```diff
@@ -1,5 +1,19 @@
 #!/bin/bash
 
+LOG_PATH=$2
+STDERR_FILE="$LOG_PATH/stderr_out"
+TMP_1="$LOG_PATH/tmp_1"
+TMP_2="$LOG_PATH/tmp_2"
+
 export PYTHONPATH=$1
-../../../omi/Unix/output/bin/omiserver -d
+../../../omi/Unix/output/bin/omiserver -d 1>$TMP_1 2>$TMP_2
```
Comment (JumpingYang001) [inline testcli/start_server]: I doubt -d will not output anything.

**Review by JumpingYang001 — APPROVED** [review]
LGTM, you might need to rebase from master:
```
git fetch
git checkout master
git pull origin master
git reset --hard HEAD
git checkout yourbranch
git rebase master
git push origin yourbranch -f
```
if you use upstream, might like:
```
git fetch upstream
git checkout yourbranch
git rebase upstream/master
git push origin yourbranch -f
```

File: testcli/start_server (hunk 1 above)
Comment (c64cosmin) [reply testcli/start_server]: @JumpingYang001 Yes, that won't output anything, removing it now.

**Comment by c64cosmin** [comment]
Removed the unneeded trace for server daemon.

**Pull request merged**